runtime/colors/blue.vim

```diff
@@ -17,7 +17,11 @@ let s:t_Co = &t_Co
 
 if (has('termguicolors') && &termguicolors) || has('gui_running')
   let g:terminal_ansi_colors = ['#000000', '#cd0000', '#00cd00', '#cdcd00', '#0000ee', '#cd00cd', '#00cdcd', '#e5e5e5', '#7f7f7f', '#ff0000', '#00ff00', '#ffff00', '#5c5cff', '#ff00ff', '#00ffff', '#ffffff']
+  for i in range(g:terminal_ansi_colors->len())
```

Comment:
```suggestion
  " Nvim uses g:terminal_color_x instead
  for i in range(g:terminal_ansi_colors->len())
    let g:terminal_color_{i} = g:terminal_ansi_colors[i]
  endfor
endif
```

Add/keep the comments (so it's easy to see where & why we've diverged from upstream) and remove the added empty lines (minimize differences). LGTM otherwise :+1: